website/src/website/pages/playground/playgroundRunner.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<meta charset="utf-8" />
+		<link href="../../../runner/style.scss" rel="stylesheet" />
```

Comment:
Ah here.

I'm not so sure what to think about the duplication of all the html files though.
Now every file has to include the styles.